ui_framework/doc_site/src/views/table/controlled_table.js

```diff
@@ -0,0 +1,214 @@
+import React from 'react';
+
+import {
+  KuiToolBar,
+  KuiToolBarSearchBox,
+  KuiToolBarSection,
+  KuiButton,
+  KuiButtonIcon,
+  KuiTable,
+  KuiToolBarText,
+  KuiControlledTable,
+  KuiTableRowCellLiner,
+  KuiPager,
+  KuiTableRow,
+  KuiTableRowCell,
+  KuiTableRowCheckBoxCell,
+  KuiTableHeaderCell,
+  KuiToolBarFooterSection,
+  KuiToolBarFooter,
+  KuiTableHeaderCheckBoxCell,
+  KuiTableBody,
+  KuiTableHeader,
+} from '../../../../components';
+
+import {
+  LEFT_ALIGNMENT,
+  RIGHT_ALIGNMENT
+} from '../../../../services';
+
+// Arbitrarily use the 3rd index in the sample data as the unique identifier for handling checkbox toggling.
+const CELL_ID_INDEX = 3;
+
+export class ControlledTable extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      selectedItems: []
```

Comment:
Chatted offline, for the sake of getting this checked in, I'll remove the toggleAll logic since we can't agree on it. I kept the individual toggle row logic so people can at least see how a row looks checked and unchecked.